recipe/meta.yaml

```diff
@@ -21,8 +21,8 @@ requirements:
     - setuptools
   run:
     - python >=3.6
-    - snowflake-connector-python <3
-    - sqlalchemy <2
+    - snowflake-connector-python <3.0.0
+    - sqlalchemy >=1.4.0,<2.0.0
```

Comment:
This bound got missed in #7 and is causing me trouble with pip check in https://github.com/conda-forge/apache-airflow-providers-snowflake-feedstock/pull/4

I'm a bit worried that the solver will just use build 0 when build 1 doesn't work.  But this is a pretty small thing to report the package as broken for.  Let's see how it goes.

Reply:
We should write a small repodata patch that fixes this. Then build 0 would also be usable.